### PR TITLE
Update dsfinv-k-generation.md

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
-Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, shows how the previously described input values are mapped by fisklatrust to the files and data of the DSFinV-K export and defines how additional, for the DSFInV-K required, values can be sent to the ft.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge). 
+Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, how the previously described input values are mapped by fiskaltrust to the files and data of the DSFinV-K export, and defines how additional, for the DSFinV-K required values can be sent to the fiskaltrust.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge).
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:8]: i do not understand the part that says: _, and defines how additional, for the DSFinV-K required values can be sent to the ..._ does it mean that there are extra values, that are additional?? so..
..and defines how values additional to the DSFinV-K required ones can be sent to..?

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+### Structure
 
 The DSFinV-K export is divided into the following sections/modules: 
 
 - Single recordings module (DE: Einzelaufzeichnungsmodul)
 - Master data module (DE: Stammdatenmodul)
 - Cashpoint closing module (DE: Kassenabschlussmodul)
 
-Each module consists of several files. In the following we will take a look at the individual modules especially the content of the files and data. Details about the meaning of the files and their individual fields can be found in the official DSFinV-K specification.
+Each module consists of several files. In the following, we will look at the individual modules, especially the files and data content. Details about the meaning of the files and their individual fields can be found in the official DSFinV-K specification.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:18]: In the following section, we will look at some (are we looking at all of them?) individual modules, especially the files and data content ones.

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md [hunk 3]
```diff
@@ -49,72 +49,72 @@ This chapter lists the data fields that are mandatory and have to be provided by
...
-#### Single recordings module (DE: Einzelaufzeichnungsmodul)
+### Single recordings module (DE: Einzelaufzeichnungsmodul)
 
 The single recordings provide the basis for data storage. These are divided into two main areas:
 
 - Bonpos (lines.csv)
 - Bonkopf (transactions.csv)
 
-In addition to these two files there are further detail files which are listed in the following. 
+In addition to these two files, there are further detail files which are listed in the following.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:61]: ..which are listed below.

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md [hunk 4]
```diff
@@ -49,72 +49,72 @@ This chapter lists the data fields that are mandatory and have to be provided by
...
+According to the DSFinV-K specification, in the data field STK_BR in the Bonpos file, either the reduced amount is displayed immediately (and the origin of the amount in the file Bonpos_Preisfindung) or the reduction in charges is displayed as a separate line item with negative amounts (with correct tax assignment; see file Bonpos_USt). The GV_TYP "Rabatt" is available for the separate line. For the reduction in charges, the current version of the fiskaltrust.Middleware expects a separate negative `ftChargeItem` with the corresponding `ftChargeItemCase` (e.g. `0x4445000000000031`) which maps to the GV_TYP "Rabatt". 
 
 
-##### File: Bonpos_Zusatzinfo (subitems.csv)
+#### File: Bonpos_Zusatzinfo (subitems.csv)
 
-This table allows to detail the composition of sold combinations of goods (DE: Warenzusammenstellungen). They serve exclusively for explanation.
+This table allows to detail the composition of sold combinations of goods (DE: Warenzusammenstellungen). They serve exclusively for an explanation.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:113]: They serve exclusively as an explanation

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md [hunk 5]
```diff
@@ -49,72 +49,72 @@ This chapter lists the data fields that are mandatory and have to be provided by
...
 
-This does not affect the basis of assessment for VAT. In the case of goods combinations with different tax rates, however, information is stored here which serves to control the distribution of the VAT assessment basis (example: Inhouse breakfast menu consisting of a coffee, a drink and buffet meal). In addition, orders deviating from the standard order can be taken into account to record the actual consumption of goods (example: gyros plate with chips instead of rice).
+This does not affect the basis of assessment for VAT. In the case of goods combinations with different tax rates, however, information is stored here which serves to control the distribution of the VAT assessment basis (for example, Inhouse breakfast menu consisting of a coffee, a drink and buffet meal). In addition, orders deviating from the standard order can be taken into account to record the actual consumption of goods (for example, gyros plate with chips instead of rice).
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:115]: stored here, (missing comma) which serves in controlling the distribution

Comment (DoxaPap) [inline doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:115]: NEVER rice with gyros, I am sorry, but there are lines which we _cannot_ cross :D :D  always french fries!

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md (hunk 1 above)
Comment (aedler-ft) [reply doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:8]: i'll try rephrase it

File: doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md (hunk 5 above)
Comment (aedler-ft) [reply doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md:115]: ;-)

**Pull request merged**